pkg/resources/helpers.go

```diff
@@ -254,19 +273,35 @@ func getTags(from interface{}) (to tags) {
 }
 
 func nestedProperty(innerType schema.ValueType, fieldDescription string) *schema.Schema {
-	return &schema.Schema{
+	return nestedPropertyWithModifiers(innerType, fieldDescription, nil, nil)
+}
+
+func nestedPropertyWithInnerModifier(innerType schema.ValueType, fieldDescription string, modifyInner func(inner *schema.Schema)) *schema.Schema {
+	return nestedPropertyWithModifiers(innerType, fieldDescription, nil, modifyInner)
+}
+
+func nestedPropertyWithModifiers(innerType schema.ValueType, fieldDescription string, modifyOuter func(outer *schema.Schema), modifyInner func(inner *schema.Schema)) *schema.Schema {
```

Comment:
what a magiuc

Reply:
Nested properties will be removed so no more 🪄